doc/source/parameters/dem/post-processing.rst

```diff
@@ -13,16 +32,14 @@ Lethe-DEM has built-in post-processing capabilities. All post-processing results
 .. important::
  The particles' average velocity and the granular temperature are cell based results calculated instantaneously at each outputting time-step. This means that they are only instantaneous spatial averages, i.e, DO NOT correspond to temporal averages measured between the outputs. For a higher resolution of the results in time, we recommend the use of a lower ``output frequency`` (:doc:`../dem/simulation_control`).
 
-The post-processing subsection of the ``.prm`` file is according to the following example:
+----------------------------
+Force chains post-processing
+----------------------------
+The ``force chains`` feature enables VTU, PTVU, and PVD files to visualize force chains between particles. Generated files's names begin with the prefix ``force_chains-``. The files include:
```

Comment:
This one uses a suffix and the previous one (see above) uses a prefix. Maybe we should uniformize this to have only suffixes?

Reply:
agree. 

Both sufix should be set by the ``output name`` parameter followed by either "force_chains" or "lagrangian_post_processing" (or something else)